Templates/BlobTrigger-CSharp/readme.md

```diff
@@ -0,0 +1,11 @@
+# BlobTrigger - C<span>#</span>
+
+The `BlobTrigger` makes it incredibly easy to react to new Blobs inside of Azure Blob Storage. This sample demostrates a simple use case of processing data from a given Blob using C#.
+
+## How it works
+
+For a `BlobTrigger` to work, you provide a path which dictates where the blobs are located inside your container, and can also help restrict the types of blobs you wish to return. For instance, you can set the path to `samples/{name}.png` to restrict the trigger to only the samples path and only blobs with ".png" at the end of their name.
```

Comment:
Might want to describe how the binding variables work (e.g. {name}), and how those captured values are available to other bindings. Not sure if/where we're describing how this works to users. Same for Queue trigger binding - we allow you to bind to any JSON properties in your message.